Lib/fontTools/ttLib/tables/_n_a_m_e.py

```diff
@@ -120,6 +120,40 @@ def getDebugName(self, nameID):
 		else:
 			return None
 
+	def getBestDebugName(self, nameID):
```

Comment:
that's not what I had in mind though. I was thinking of a more generic method like this one:

```python
def getBestDebugName(self, nameIDs):
    for nameID in nameIDs:
        name = self.getDebugName(nameID)
        if name is not None:
            return name
    return None
```

Reply:
though 'best' is not the _best_ way to name this; it's basically just doing `getFirstDebugName`

Reply:
@anthrotype Oh, sorry. My bad. I changed it. Thanks a lot.

Reply:
@anthrotype  So, should I rename it from getBestDebugName to getFirstDebugName ?

Reply:
yes, I think it describes better the intent